deploy/local_single_user_deploy.sh

```diff
@@ -208,28 +316,37 @@ stop() {
     echo -e "${DEBUG}All processes stopped${NC}"
 }
 
-
-
 # Usage function with new options
 usage() {
     echo "Usage: $0 [options]"
     echo "Options:"
-    echo "  -h, --help             Display this help and exit"
-    echo "  -n, --num-keepers NUM  Set the number of keeper processes"
-    echo "  -w, --work-dir DIR     Set the working directory"
-    echo "  -i, --install          Install all components"
-    echo "  -r, --reset            Reset all components"
-    echo "  -k, --kill             Kill all components"
-    echo "  -s, --stop             Stop all components"
+    echo "  -h|--help       Display this help and exit"
```

Comment:
pls consider consistent alignment. the first six are aligned, but remaining are not.